### ISSUE TITLE
pywemo virtual device error message

### ISSUE THREAD

**Issue by MatGriffin** [rudders/homebridge-wemo#31]
Log repeatedly reports reports

[Wemo] [pywemo virtual device] [urn:Belkin:device:switch:1] is unsupported but feel free to create a GitHub issue.

### Details of your setup.
(1) Homebridge UI-X

* Which version of Homebridge/HOOBS do you have?
1.3.4

* Which platform do you run Homebridge/HOOBS on
Raspberry Pi
Node.js Version | v14.17.5
Npm v6.14.14

* Which version of this plugin (homebridge-wemo) do you have? Has the issue started since upgrading from a previous version?
Move from old version 3.x to v4.1.3 and now have this

Which Wemo devices do you have that are causing issues? Please include product models if applicable.
4 first edition UK sockets
1 Wemo Maker

```
[Wemo] [pywemo virtual device] [urn:Belkin:device:switch:1] is unsupported but feel free to create a GitHub issue.
```

**Comment by bwp91** [comment]
Hi,

I have never seen this device type before [urn:Belkin:device:switch:1], is it an emulated switch?

Do your physical devices show in homekit and function?

**Comment by MatGriffin** [comment]
Yes, all physical devices showing and working. I don't think there are any emulated, I'll double check though

**Comment by bwp91** [comment]
It's definitely something to do with the [pywemo](https://github.com/pywemo/pywemo) library. So whether you knowingly or unknowingly make use of this somewhere on your network?

It looks like I could make it appear as a switch in homebridge, but I don't know whether it has any use? That's why I asked if you have like a virtual switch setup with this library? Or I can just get the plugin to ignore this device type so the log messages don't appear.

**Comment by bwp91** [comment]
I've gone ahead and released a beta version of the plugin, 4.1.4-beta.0, which should ignore this device completely. When you get the chance could you try it out just to confirm that it's not displaying anymore logs? Thanks

**Comment by MatGriffin** [comment]
Thanks. I  have updated to the beta, I'm no longer seeing the message,

**Comment by bwp91** [comment]
Thanks, just released v4.1.4 with the change
